### ISSUE TITLE
Switching to use JSON CLDR data instead of java

### ISSUE THREAD

**Issue by caridy** [andyearnshaw/Intl.js#86]
I have been working on this for a while now to simplify the build process, similarly to what we did for formatjs libraries. As part of the process, I have an implementation that matches the current process, producing almost the exact same data (updating few things since I'm using the latest cldr data).

One thing that is problematic is that fact that JSON CLDR only contains the common ("stable"), it does not contains the seeds ("alpha"), this means there are few locales that will be dropped if we make the switch today. Here is the list:

```
wal-ET
wal
xh-ZA
xh
tg-Cyrl-TJ
tg-Cyrl
tg
tig-ER
tig
st-LS
st-ZA
st
gv-GB
bm-ML
byn-ER
byn
```

**Comment by caridy** [comment]
Checking support for those 16 locales in browser using this code:

```js
["wal-ET", "wal", "xh-ZA", "xh", "tg-Cyrl-TJ", "tg-Cyrl", "tg", "tig-ER", "tig", "st-LS", "st-ZA", "st", "gv-GB", "bm-ML", "byn-ER", "byn"].forEach(function (locale) {
    console.log((new Intl.DateTimeFormat('en')).resolvedOptions().locale);
});
```

None of those 16 locales is supported in FF, and only `tg` is supported in chrome.

**Comment by caridy** [comment]
This makes it even more compelling the story to remove those unstable locales from the polyfill. /cc @ericf

**Comment by ericf** [comment]
So for these locales, they are different than the `root` locale, and there's no overlapping browser support. And in the CLDR JSON data these locales have zero data that the polyfill needs.

It seems like something is missing here since the output is different between these locales when running them in IE — where is this data coming from if not the CLDR?

@bterlson do you know where Windows (especially 10) is getting its data to back the `Intl` APIs? The locales that @caridy mentioned above don't have number or calendar data in the CLDR JSON dist.

**Comment by andyearnshaw** [comment]
The fact that they are missing from browsers could be seen as a compelling reason to keep them (so people can polyfill support–albeit unstable support–for those languages).  However, I don't really see dropping them as a problem.  Anybody who really needs them could probably just generate the JSON files themselves.

**Comment by caridy** [comment]
@ericf JSON CLDR only contains the common (stable locales), while LDML has all the data. The common ("stable") unicode.org/repos/cldr/trunk/common/ and the seed ("alpha") http://unicode.org/repos/cldr/trunk/seed/main/

**Comment by ericf** [comment]
@caridy okay, that explains what we're seeing then. Are there docs anywhere that describe what the seed/alpha set of locales means?

**Comment by bterlson** [comment]
Chakra uses Windows APIs to carry out localization tasks, and so uses Windows' built-in locale data.

**Comment by caridy** [comment]
https://github.com/andyearnshaw/Intl.js/pull/85

**Comment by caridy** [comment]
the new data structure is available in intl@1.0.0-rc-1
